db/migrate/20151114125209_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
```

Comment:
I tried removing the `slug` fields on each table and that didn't work. Then I tried removing the full `friendly_id_slugs` table and it still works. I think that table is only needed if you need to keep track of the previously slugs used for records.